vm/devices/net/netvsp/src/lib.rs

```diff
@@ -147,11 +147,19 @@ const LINK_DELAY_DURATION: Duration = Duration::from_secs(5);
 #[cfg(test)]
 const LINK_DELAY_DURATION: Duration = Duration::from_millis(333);
 
-#[derive(PartialEq)]
-enum CoordinatorMessage {
+#[derive(Default, PartialEq)]
+struct CoordinatorMessageUpdateType {
     /// Update guest VF state based on current availability and the guest VF state tracked by the primary channel.
     /// This includes adding the guest VF device and switching the data path.
-    UpdateGuestVfState,
+    guest_vf_state: bool,
+    /// Update the receive filter for all channels.
+    filter_state: bool,
```

Comment:
i.e., maybe this can be `Option<u32>` so that we can store the packet filter in the coordinator itself.